src/lib/components/sections/navbar.svelte

```diff
@@ -31,44 +31,46 @@
   <div class="container">
     <div class="logo-container">
       <a href="/" class="logo">
-        <img src="assets/badges/general.svg" class="badge" alt="acmCSUF logo" />
-        <h3>CSUF</h3>
+        <img src="assets/badges/general.svg" class="badge" alt="acmCSUF badge" />
+        <h3 class="brand-header">at <b>CSUF</b></h3>
```

Comment:
Bold tag works the same functionally as CSS .brand-em. Might depreciate that CSS style.